cypress/integration/check_summary/summary-count-after-delete.spec.js

```diff
@@ -0,0 +1,55 @@
+/// <reference types="cypress" />
+
+Cypress.on('uncaught:exception', (err, runnable) => {
+  // returning false here prevents Cypress from
+  // failing the test
+  return false
+})
+
+describe('Testing summary count after deleting one of the endpoints', () => {
```

Comment:
In other steps of the test, we just get the count of the enabled summary and assign it to a variable.
This test is just to make sure the values are the same after deleting one of the endpoints and checking on other steps doesn't add any value because we have to check if a value equals itself.